.nvmrc

```diff
@@ -0,0 +1 @@
+10.10.0
```

Comment:
My questions are:
- If your nvm has v11, does it change to v10.10.0? (not necessary)
- Is it enough to only put `"engine"` in package.json?

Reply:
You're right. I'll remove the .nvmrc file. It's more useless than I thought

If I'm using the wrong Node.js version it warns me when I run `yarn` as follows:
```
scon @ ~/code/src/paritytech/js-libs - [luke-81-electron-4] $ nvm use 10.6.0
Now using node v10.6.0 (npm v6.1.0)
scon @ ~/code/src/paritytech/js-libs - [luke-81-electron-4] $ yarn
yarn install v1.12.3
[1/5] 🔍  Validating package.json...
error js-libs@3.0.26: The engine "node" is incompatible with this module. Expected version ">=10.10.0". Got "10.6.0"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

You're right, there's just as many steps whether we do or don't have the .nvmrc file. The important thing is to have the "engine" specified.

```
scon @ ~/code/src/paritytech/js-libs - [luke-81-electron-4] $ node --version
v10.6.0
scon @ ~/code/src/paritytech/js-libs - [luke-81-electron-4] $ nvm use
No .nvmrc file found
```

With .nvmrc file:
```
scon @ ~/code/src/paritytech/js-libs - [luke-81-electron-4] $ node --version
v10.6.0
scon @ ~/code/src/paritytech/js-libs - [luke-81-electron-4] $ nvm use
Found '/Users/scon/code/src/paritytech/js-libs/.nvmrc' with version <10.10.0>
N/A: version "10.10.0 -> N/A" is not yet installed.

You need to run "nvm install 10.10.0" to install it before using it.
scon @ ~/code/src/paritytech/js-libs - [luke-81-electron-4] $
```